GooseMPL/__init__.py

```diff
@@ -1969,25 +1997,81 @@ def hist(P, edges, **kwargs):
     return p
 
 
-def cdf(data, mode="continuous", **kwargs):
+def ccdf(data: ArrayLike):
     """
-    Return cumulative density.
+    Complementary cumulative distribution function: ``P(x >= X)``
 
-    :arguments:
+    :param data: Input data.
+    :return: ``(P, X)``
+    """
 
-        **data** (``<numpy.ndarray>``)
-            Input data, to plot the distribution for.
+    bin_edges = np.unique(data)
+    ibin = np.digitize(data, bin_edges[::-1], right=True)
+    _, count = np.unique(ibin, return_counts=True)
```

Comment:
I was worried that I misinterpreted the output of `digitize` (counter starts at `1` not at `0`), but I don't think that that matters